### PR TITLE
Ssh: add Match keyword support

### PR DESCRIPTION
**Pull request by granquet**
Signed-off-by: Guillaume Ranquet <guillaume-externe.ranquet@edf.fr>

Proposal for a fix regarding https://github.com/hercules-team/augeas/issues/632

### CODE REVIEW

File: lenses/ssh.aug [hunk 1]
```diff
@@ -105,11 +105,26 @@ module Ssh =
...
+     [ label "Condition" . condition_entry+ . eol ]
+
+   let match_entry = entry
+
+   let match =
+     [ key /Match/i . match_cond
+        . [ label "Settings" .  match_entry+ ]
+     ]
+
+
```
Comment (raphink) [inline lenses/ssh.aug:122]: Or maybe just `let match = Sshd.match` so we reuse the logic from the other lens. What do you think @lutter?

Comment (granquet) [reply lenses/ssh.aug:122]: It doesn't seem to work... the definition of eol in ssh.aug and sshd.aug are different... an induce a change in condition_entry ( [^ \t\r\n] vs [^ \t\n])
not sure how to solve that as I'm just discovering augeas... aside from trying to rewrite ssh.aug and sshd.aug.

Comment (raphink) [reply lenses/ssh.aug:122]: Ok then. Thanks for trying it!

**Review by raphink — APPROVED** [review]
Lgtm

**Pull request merged**